BDArmory/Armor/BDAdjustableArmor.cs

```diff
@@ -34,78 +38,49 @@ public class BDAdjustableArmor : PartModule
         public string ThicknessTransformName = "ThicknessTransform"; //name of armature to control thickness of curved panels
         Transform ThicknessTransform;
 
-        AttachNode N1;
-        AttachNode N2;
-        AttachNode N3;
-        AttachNode N4;
+        [KSPField] public string stackNodePosition;
```

Comment:
Basically completely rebuilt from the old kludge job into something that actually works